guides/release/using-typescript/application-development/converting-an-app.md

```diff
@@ -0,0 +1,105 @@
+These directions are for converting an _existing_ Ember app to TypeScript. If you are starting a new app, you can use the directions in [Getting Started].
+
+## Enable TypeScript Features
+
+### A Bit of a Hack
+
+Since `ember-cli` _currently_ has no flag to convert your project to TypeScript, we're going to use a bit of a hack and _temporarily_ install the legacy `ember-cli-typescript` addon to complete most of the migration:
```

Comment:
This approach kinda stinks but I experimented w/ different ways and this was the least obnoxious one. I have it on my TODO list to add an appropriate flag...probably to ember-cli-update.